doc/tutorials/imgproc/erosion_dilatation/erosion_dilatation.markdown

```diff
@@ -143,3 +206,33 @@ naturally. Try them out! You can even try to add a third Trackbar to control the
 iterations.
 
 ![](images/Morphology_1_Result.jpg)
+
+Troubleshooting
+---
+@add_toggle_python
```

Comment:
@alalek I know, that was my first thought too. However, I confronted a lot of issues while installing OpenCV for python under Ubuntu. I also have trouble for the java installation. The sites and tutorials found in docs and online are very error prune or are for older versions. Some tutorials aren't working at all or install OpenCV and the version is shown but nothing more (executing example code throws errors). That's why I though it would be useful to bring some clarification as a "troubleshooting" section. But of course I can remove the section. 

What's your opinion about the other content (except the troubleshooting section now)? 

P.S. I want to start also a discussion somewhere about simplifying the installation process by collecting some opinions and discuss various installation options and the problems beginners have while starting learning OpenCV. That way the troubleshooting section won't be necessary at all and the beginners can get started much faster. My experience for example wasn't that great and the installation process was too complicated for "just trying it out". Finding the right resources and explanations was quite difficult.

Reply:
I believe this content should be contributed into this document instead: https://github.com/opencv/opencv/wiki/FAQ (maintained by @asmorkalov )
Please open a new issue with proposals for Wiki updates (GitHub doesn't support PRs for Wiki)

There is PR with updating of installation process: #18195
Feel free to discuss it there.

Reply:
@asmorkalov do you think we can fit somehow this kind of content in the FAQ?

And about the mentioned PR, I'll check it out, thanks. Looks like a lot of improvements are being made.